src/FontMetrics.php

```diff
@@ -65,6 +66,18 @@ class FontMetrics
      */
     private $options;
 
+    /**
+     * Character mapping table for fonts
+     * @var array
+     */
+    private $unicodeCharMapTables = [];
```

Comment:
Both properties are unused.

Reply:
Thanks, I missed these when I was cleaning up from previous iterations of the logic.